src/core/lib/security/authorization/rbac_translator.cc

```diff
@@ -373,7 +369,8 @@ absl::StatusOr<RbacPolicies> GenerateRbacPolicies(
   }
   absl::string_view name = it->second.string_value();
   RbacPolicies rbacs;
-  std::unique_ptr<Rbac> allow_rbac;
+  Rbac allow_rbac;
```

Comment:
This temp variable isn't needed.  Instead, you can just directly set `rbacs.allow_policy`.

Reply:
Done.